model_analyzer/device/gpu_device_factory.py

```diff
@@ -223,19 +226,28 @@ def get_cuda_visible_gpus(self):
             UUIDs of the DCGM supported devices visible to CUDA
         """
 
-        cuda_visible_gpus = []
-        if numba.cuda.is_available():
-            for cuda_device in numba.cuda.list_devices():
-                try:
-                    cuda_visible_gpus.append(
-                        self.get_device_by_cuda_index(cuda_device.id))
```

Comment:
it seems that this function should be removed if it is no longer used?

Reply:
The function is still being used.